packages/cli/src/commands/generate/forms/generate_forms_command.ts

```diff
@@ -72,6 +72,8 @@ export class GenerateFormsCommand
     const backendOutputClient = this.backendOutputClientBuilder();
 
     let output;
+    const credentialsErrorMessage =
```

Comment:
It doesn't seem we should share this message.

Should we have?
`Unable to get backend outputs due to invalid credentials.`
and
`Unable to get backend outputs due to insufficient permissions.`

i.e. one of these cases is: credentials are ok, but they lack access grants.